src/components/input/input.scss

```diff
@@ -51,6 +52,38 @@
     text-indent: sizes.$control-text-inset;
   }
 
+  /**
+   * Multi-line types should be taller
+   */
+
+  @at-root textarea#{&} {
```

Comment:
Nit: I find this harder to read than the previous `textarea.c-input`. Not a blocker by any means though

Reply:
I agree, but I wanted all of the type-specific CSS to be together so the various states would apply at the same point in the cascade.